lib/queuePopulator/IngestionPopulator.js

```diff
@@ -239,21 +239,28 @@ class IngestionPopulator {
     }
 
     _subscribeToLogSourceDispatcher(zkEndpoint, bucketd, cb) {
```

Comment:
What is `cb` used for at the end, by the caller, and isn't it possible to remove `cb` if so? I think it could help reduce confusion as it's not a regular callback, the `subscribeToLogSourceDispatcher` logic is essentially synchronous as it only kicks off the provisioning, it does not wait for something afterwards.

Reply:
ah ok, sorry I was so fixated on keeping it async somehow because of the previous methods, but I see what you mean. No need to pass `cb` here and just make it a sync call that will update the readers when provisioned